AzureDevOpsPowerShell/Public/Api/Core/Projects/New-AzDoProject.ps1

```diff
@@ -149,8 +149,15 @@ function New-AzDoProject {
       }
 
       if ($PSCmdlet.ShouldProcess($CollectionUri, "Create project named: $($PSStyle.Bold)$name$($PSStyle.Reset)")) {
-
-        $body | Invoke-AzDoRestMethod @params | Out-Null
+        try {
+          $body | Invoke-AzDoRestMethod @params | Out-Null
+        } catch {
+          if ($_ -match 'TF200019') {

```

Comment:
How do I read this, a duplicate ProjectName POST-operation will throw TF200019 and if this is the case it'll continue, any other terminating will write an error-record?

Reply:
yes